### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by SegFault42**
small change in Readme.md

### CODE REVIEW

**Comment by skyl** [comment]
Prefer Python3 syntax for print. But, aside from that, I don't think print() should appear here. There could be a line in between explaining that the object is returned.

So, instead of:

```python
from bittrex.bittrex import *

my_bittrex = Bittrex("<my_api_key>", "<my_api_secret>", api_version="<API_V1_1> or <API_V2_0>")

my_bittrex.get_balance('ETH')
{'success': True, 
 'message': '',
 'result': {'Currency': 'ETH', 'Balance': 0.0, 'Available': 0.0, 
            'Pending': 0.0, 'CryptoAddress': None}
}
```

It could be broken into 2 like:

```python
from bittrex import Bittrex
my_bittrex = Bittrex("<my_api_key>", "<my_api_secret>", api_version="<API_V1_1> or <API_V2_0>")
my_bittrex.get_balance('ETH')
```

This call to `get_balance` returns an object such as the following:

```
{'success': True, 
 'message': '',
 'result': {'Currency': 'ETH', 'Balance': 0.0, 'Available': 0.0, 
            'Pending': 0.0, 'CryptoAddress': None}
}
```

**Comment by SegFault42** [comment]
updated :+1:

**Comment by skyl** [comment]
Comment should appear outside of the code block: https://github.com/SegFault42/python-bittrex/blob/123c1ab52b06738df52c9845c0946c0e4c8e244d/README.md

File: README.md [hunk 1]
```diff
@@ -39,6 +43,11 @@ from bittrex.bittrex import *
 my_bittrex = Bittrex("<my_api_key>", "<my_api_secret>", api_version="<API_V1_1> or <API_V2_0>")
 
 my_bittrex.get_balance('ETH')
+
+```
+```
+This call to get_balance returns an object such as the following: :
```
Comment (skyl) [inline README.md]: extra colon.

File: README.md [hunk 2]
```diff
@@ -34,9 +34,12 @@ from bittrex.bittrex import Bittrex, API_V2_0
 
 my_bittrex = Bittrex(None, None, api_version=API_V2_0)  # or defaulting to v1.1 as Bittrex(None, None)
 my_bittrex.get_markets()
-{'success': True, 'message': '', 'result': [{'MarketCurrency': 'LTC', ...
 ```
 
+This call to get_markets returns an object such as the following:
+
+{'success': True, 'message': '', 'result': [{'MarketCurrency': 'LTC', ...
```
Comment (skyl) [inline README.md]: The narrative text should not be wrapped as a code block.

```
{'but': 'the JSON should be'}
```

**Comment by skyl** [comment]
LGTM

**Pull request merged**